nexus-repository-r-it/pom.xml

```diff
@@ -88,6 +88,17 @@
       <version>${nxrm-version}</version>
       <type>zip</type>
       <scope>test</scope>
+      <exclusions>
+        <!-- Avoid leaking older r plugins from distro -->
+        <exclusion>
+          <groupId>org.sonatype.nexus.plugins</groupId>
+          <artifactId>nexus-repository-r</artifactId>
```

Comment:
This change made me wonder if we should update the nexus-format-archetype with a similar change? 

If so, please have a look at this PR: https://github.com/sonatype-nexus-community/nexus-format-archetype/pull/37